app/assets/javascripts/avalon.js

```diff
@@ -47,4 +47,6 @@ $(document).ready(function() {
     })
     $('input[readonly], textarea[readonly]').attr("readonly", false);
   }
+
+  $('a').has('img, ul').addClass('block');
```

Comment:
I don't understand why you're adding the class here in the javascript versus adding the class in the markup.  I can think of pros and cons to each approach but would favor less magic so I would prefer it in the html.  What are other people's thoughts, @mbklein @lealeelu @bkeese ?